Assets/World/Units/Ships/Merchants/Huker/Huker.gd

```diff
@@ -0,0 +1,12 @@
+tool
+extends Ship
+class_name Huker
+
+onready var faction_color: Sprite3D = $FactionColor as Sprite3D
+
+func update_faction_color() -> void:
+	if faction_color != null:
+		var mat = Global.COLOR_MATERIAL[faction] as ShaderMaterial
+		var sm = faction_color.set_material_override(mat)
```

Comment:
This line still has problems. `set_material_override` does not return a value, and you're not using `sm` anywhere. Remove the declaration so that it looks like this:

    faction_color.set_material_override(mat)